test/cmd/deployments.sh

```diff
@@ -66,6 +66,20 @@ os::cmd::expect_success_and_not_text 'oc env dc/test-deployment-config --list' '
 echo "env: ok"
 os::test::junit::declare_suite_end
 
+os::test::junit::declare_suite_start "cmd/deployments/retry"
+os::cmd::expect_success_and_text 'oc run retry-test --image=openshift/origin-pod' 'created'
+os::cmd::expect_success_and_text 'oc rollout status dc/retry-test' 'has completed progressing'
```

Comment:
I don't think the rollout will ever complete since we don't run Docker in these tests.